experiments/react-native-uifabric/src/RNTester/TestComponents/ThemeTest.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import * as ReactNative from 'react-native';
+import { ThemeProvider, useTheme } from '@uifabricshared/theming-react-native';
+import { commonTestStyles } from '../styles';
+import { Button, PrimaryButton, StealthButton, Text } from '../../components';
+
+const Panel: React.FunctionComponent = () => {
+    const [disabled, setDisabled] = React.useState(false);
+    const onClick = React.useCallback(() => setDisabled(!disabled), [disabled, setDisabled]);

```

Comment:
Is memoizing on `disabled` not sufficient?

Reply:
This code is moved as-is from the existing code in `FabricTest.tsx`, line 40. I'm just moving it out to a new file.